### PR TITLE
Add libdef for metalsmith-plugins

### PR DESCRIPTION
**Pull request by jirutka**
https://github.com/ismay/metalsmith-layouts

### CODE REVIEW

**Comment by jirutka** [comment]
How can I declare dependency on a type defined in another libdef?

**Comment by AndrewSouthpaw** [comment]
You can't, there's more discussion in `CONTRIBUTING.md`.

File: definitions/npm/metalsmith-layouts_v2.x.x/flow_v0.75.x-/metalsmith-layouts_v2.x.x.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-unused-vars, no-redeclare */
+
+declare module 'metalsmith-layouts' {
+
+  import type { Plugin } from 'metalsmith';
```
Comment (AndrewSouthpaw) [inline definitions/npm/metalsmith-layouts_v2.x.x/flow_v0.75.x-/metalsmith-layouts_v2.x.x.js]: This won't work, discussed in contributing doc.

File: definitions/npm/metalsmith-layouts_v2.x.x/flow_v0.75.x-/test_metalsmith-layouts.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  }
+});
+
+const plugin: Plugin = layouts({
+  default: 'base.html.hbs',
+  directory: './layouts',
+  pattern: ['**/*.html', '**/*.htm']
+});
+
+// $ExpectError
```
Comment (AndrewSouthpaw) [inline definitions/npm/metalsmith-layouts_v2.x.x/flow_v0.75.x-/test_metalsmith-layouts.js]: Use `describe`/`it` blocks

File: definitions/npm/metalsmith-layouts_v2.x.x/flow_v0.75.x-/metalsmith-layouts_v2.x.x.js (hunk 1 above)
Comment (jirutka) [reply definitions/npm/metalsmith-layouts_v2.x.x/flow_v0.75.x-/metalsmith-layouts_v2.x.x.js]: I see now. flow-typed still does not support importing type definitions from other libdefs. That's really silly, I'd need to copy&paste basically complete type definitions file from metalsmith to write typedefs for metalsmith's plugins. I don't wanna do that.

Comment (AndrewSouthpaw) [reply definitions/npm/metalsmith-layouts_v2.x.x/flow_v0.75.x-/metalsmith-layouts_v2.x.x.js]: Yeah, we're all pretty bummed about it. #flowStruggles.

**Pull request closed without merging**